endpoint.go

```diff
@@ -600,7 +606,23 @@ func (ep *endpoint) sbLeave(sbox Sandbox, options ...EndpointOption) error {
 	return sb.clearDefaultGW()
 }
 
-func (ep *endpoint) Delete() error {
+func (n *network) validateForceDelete(locator string) error {
+	if n.Scope() == datastore.LocalScope {
+		return nil
+	}
+
+	if locator == "" {
+		return fmt.Errorf("invalid endpoint locator identifier")
+	}
+
+	if n.getController().isNodeAlive(locator) {
+		return fmt.Errorf("host alive : %s", locator)
```

Comment:
Can you please make this error message more clearer for the user? Some thing like `the remote host %s where the connection is active is alive`? Also please remove the `:` and add it next to where you say `host`